modules/hmmer/eslreformat/main.nf

```diff
@@ -0,0 +1,38 @@
+process HMMER_ESLREFORMAT {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::hmmer=3.3.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/hmmer:3.3.2--h1b792b2_1':
+        'quay.io/biocontainers/hmmer:3.3.2--h1b792b2_1' }"
+
+    input:
+    tuple val(meta), path(seqfile)
+    val   format
```

Comment:
Inputs should only be used for actual file inputs. You can specify the `format` in the `nextflow.config` file using `ext.args` like: https://github.com/nf-core/modules/blob/b5e3d16d80abfc6aa1c86ffde8c1b507bea80f0d/tests/modules/bcftools/convert/nextflow.config#L9-L11

Reply:
OK, makes sense. I thought this was only for "real" params with minuses...